crates/api_common/src/utils.rs

```diff
@@ -1025,6 +1022,17 @@ pub async fn proxy_image_link_opt_apub(
   }
 }
 
+fn build_proxied_image_url(
+  link: &Url,
+  protocol_and_hostname: &str,
+) -> Result<Url, url::ParseError> {
+  Url::parse(&format!(
+    "{}/api/v3/image_proxy?url={}",
+    protocol_and_hostname,
+    encode(link.as_str())
+  ))
+}
```

Comment:
Why put this in a function when its only called once?

Reply:
mmmk, I can move it into the function.